projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts

```diff
@@ -22,29 +22,29 @@
  * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { ShareDataRow } from '@alfresco/adf-content-services';
-import { ChangeDetectorRef, Component, Input, ViewEncapsulation } from '@angular/core';
+import { ChangeDetectorRef, Component, Input, OnInit, ViewEncapsulation } from '@angular/core';
 
 @Component({
   selector: 'aca-tags-column',
   template: `
-    <adf-tag-node-list [showDelete]="false" [limitTagsDisplayed]="true" [nodeId]="getNodeId(context.row)" (results)="onTagsLoaded()">
-    </adf-tag-node-list>
+    <adf-tag-node-list [showDelete]="false" [limitTagsDisplayed]="true" [nodeId]="nodeId" (results)="onTagsLoaded()"> </adf-tag-node-list>
   `,
   styleUrls: ['./tags-column.component.scss'],
   encapsulation: ViewEncapsulation.None,
   host: {
     class: 'adf-datatable-content-cell aca-tags-name-column'
   }
 })
-export class TagsColumnComponent {
+export class TagsColumnComponent implements OnInit {
   @Input()
   context: any;
 
+  nodeId: string;
+
   constructor(private cd: ChangeDetectorRef) {}
 
-  getNodeId(row: ShareDataRow): string {
-    return row.id;
+  ngOnInit(): void {
+    this.nodeId = this.context?.row?.id;
```

Comment:
Can there be situation that context will be changed by some action in parent? If yes then this won't be updated because ngOnInit is called only once during initialization